src/post/index.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+class Post extends React.Component {
+
+    render() {
+        const { post } = this.props;
+        const { photoUrl, caption } = post;
+
+        return (
+            <>
+                <Photo photoUrl={photoUrl} />
+                <Caption  caption={caption} />
```

Comment:
Delete the extra space between `Caption` and `caption={caption}`

Reply:
@justinyum98  is on fire! 🔥